monai/networks/blocks/dynunet_block.py

```diff
@@ -31,9 +32,8 @@ class UnetResBlock(nn.Module):
         out_channels: number of output channels.
         kernel_size: convolution kernel size.
         stride: convolution stride.
-        norm_name: [``"batch"``, ``"instance"``, ``"group"``]
-            feature normalization type and arguments. In this module, if using ``"group"``,
-            `in_channels` should be divisible by 16 (default value for ``num_groups``).
+        norm_name: feature normalization type and arguments.
```

Comment:
Should we still keep the doc-string for users: "if using ``"group"``, `in_channels` should be divisible by 16 (default value for ``num_groups``)."?

Reply:
Now we leave the parameter for users to type, and we do not have the ``num_groups `` parameter in the class, thus I suggest remove this part.